frontend/src/v5/store/tickets/card/ticketsCard.redux.ts

```diff
@@ -45,6 +46,7 @@ export interface ITicketsCardState {
 	view: TicketsCardViews,
 	readOnly: boolean,
 	overrides: OverridesDicts | null,
+	unsavedTicket: IUnsavedTicket | null,
```

Comment:
is **IUnsavedTicket** type necessary? cant we we just blank the ticket when is saved and use the passed **templateId** and **containerOrFederation** for saving the ticket? I find that duplicating known data always leads to errors in a near future

Reply:
I'm not sure I understand what you mean

Reply:
You could just use an unsavedTicket: ITicket | null. Why do you have to keep the templateId and containerOrFederation in unSavedTicket?